packages/block-library/src/list/edit.js

```diff
@@ -34,7 +121,8 @@ export default function ListEdit( {
 			__unstableOnSplitMiddle={ () => createBlock( 'core/paragraph' ) }
 			onReplace={ onReplace }
 			onRemove={ () => onReplace( [] ) }
-			onTagNameChange={ ( tag ) => setAttributes( { ordered: tag === 'ol' } ) }
-		/>
+		>
+			{ controls }
```

Comment:
Do all controls need to be nested in the `RichText` component or only keyboard shortcuts? It would be helpful to add an inline comment explaining why it has to be handled this way so other maintainers don't refactor it by accident.

Reply:
Will move this to a separate PR. Everything needs to be nested so the controls have access to the RichText value.

Reply:
This is what I assumed here, I'm glad you confirmed it.